.github/workflows/release.yml

```diff
@@ -0,0 +1,131 @@
+name: release binaries
+on: 
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  build:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        mode: [osx-x64,linux-x64,win-x64]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 6.0.x
+    - name: Setup NuGet.exe for use with actions
+      uses: NuGet/setup-nuget@v1.0.6
+    - name: Restore dependencies using Nuget
+      run: nuget restore src\client.sln -Verbosity Normal -NonInteractive -ConfigFile src/nuget.config
+    - name: dotnet Publish for ${{ matrix.mode }}
+      run: dotnet publish src\dsc\dsc.csproj -c Release -r ${{ matrix.mode }} --no-restore --self-contained true --verbosity normal
+    - name: Build endpointmanagerlauncher
+      run: dotnet publish src\EndpointManagerLauncher\endpointmanagerlauncher.csproj -r win-x64 -c Release --no-restore
+    - name: Copy endpointmanagerlauncher
+      uses: Azure/powershell@v1
+      if: ${{ matrix.mode == 'win-x64' }}
+      with:
+        inlineScript: |
+          Copy-Item -Path ${{ github.workspace }}\src\EndpointManagerLauncher\bin\Release\netcoreapp3.1\win-x64\publish\ -Destination ${{ github.workspace }}\src\dsc\bin\Release\netcoreapp3.1\win-x64\publish\EndpointManagerLauncher -Recurse -Exclude **/*.pdb   
+        azPSVersion: '3.1.0'
+    - name: Download Kubectl
+      uses: Azure/powershell@v1
+      with:
+        inlineScript: |
+          if ('${{ matrix.mode }}' -eq 'win-x64') {
```

Comment:
💡 Cool, you could check this `setup-kubectl` - `Azure` GH action out :) https://github.com/Azure/setup-kubectl  More kind of Action based.

Reply:
@Tatsinnit thank you for the suggestion, I know this GH action exists and checked it as well, but this downloads the kubectl library to the runner, two things we need the kubectl to be downloaded to specific path and also for windows it needs to download from different URL/blob, and it will be .exe file. potential improvement for this GH action which we can do as an enhancement and change it later. do you agree? I will check with Oliver as well on this.

Reply:
@Tatsinnit ignore my comment above, just found out this GH action takes care of downloading kubectl based on the OS I didn't know at the first place, I will update the code. Thanks. :) do you think I can do it as separate PR?

Reply:
But this job runs on windows-latest and downloads kubectl bundled with other dotnet libraries, bridge libraries for all three OS (linux, windows, OSX) in parallel. setup-kubectl GHA wouldn't do that, it auto detects the OS and downloads only for them. @OliverMKing this would be an enhancement which can be done later.